Sources/ApolloCodegenLib/Templates/OperationIdentifiersTemplate.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+import ApolloUtils
+
+/// Provides the format to output a file used for APQ registration.
+struct OperationIdentifiersTemplate {
```

Comment:
I debated simply using `Codable` on the `OperationIdentifier` struct but it resulted in needing to write a custom encoder/decoder which at that point our templating system is quicker.